pkg/mimir/mimir.go

```diff
@@ -1019,6 +1021,14 @@ func (t *Mimir) Run() error {
 		t.API.RegisterIngesterRing(t.Ingester.RingHandler())
 	}
 
+	// Similar to above, register the querier ring handler if it exists,
```

Comment:
Nits, I don't think the "Similar to above" give any relevant information to what we are going to do below here. Also when saying "above" that might be get outdated as the code get reordered or refactored in the future. Hence I suggest to just remove the reference.

```suggestion
	// Register the querier ring handler if it exists,
```